DqtNotifications/.gitignore

```diff
@@ -399,4 +399,4 @@ FodyWeavers.xsd
 
 .idea
 
-src/DqtReporting.ConfigureSolution/Properties/launchSettings.json
\ No newline at end of file
+local.settings.json
```

Comment:
We still need `src/DqtReporting.ConfigureSolution/Properties/launchSettings.json` here

Reply:
Fixed